templates/simple/generic-barebox-tftp-ramdisk-simple-template.jinja2

```diff
@@ -0,0 +1,7 @@
+{% extends 'boot/generic-barebox-tftp-ramdisk-boot-template.jinja2' %}
```

Comment:
This test plan is actually not used on any `barebox` platform.  So maybe we can leave it out?  It doesn't really hurt, it's just a bit stale.  I think the `sleep` test plan would be more useful than `simple` btw.

Reply:
I will remove simple and add sleep